assets/js/app.tsx

```diff
@@ -35,6 +35,9 @@ if (sentryDsn) {
   Sentry.init({
     dsn: sentryDsn,
     environment: environment,
+    integrations: [Sentry.replayIntegration()],
+    replaysSessionSampleRate: 0.1,
```

Comment:
Is there value in recording sessions without errors? If we're currently just adding this in support of error tracking, maybe we don't need to.

Reply:
Yeah I suppose not. I just remembered that we are using Fullstory for full sessions. Devs would only need sessions for errors.

Reply:
Hm, I wonder if this would be capable of replacing current usage of Fullstory, or if they have different limitations.

Reply:
The issue with Fullstory is the very limited number of seats. AFAIK, only PM and Design get them. I wasn't even given a temporary one while I was implementing FS. I know there is a FS/Sentry integration, but that gives Sentry issues a link to the FS replay where the error occurred. Because we don't have seats, that doesn't really benefit us.

Reply:
> I wonder if this would be capable of replacing current usage of Fullstory

In other words, using Sentry's own session recording for whatever tasks we currently use Fullstory for (not the other way around). But this would require that A) Sentry's session recording has similar capabilities as Fullstory, and B) everyone currently using Fullstory on the project could be given Sentry access.